MANIFEST.in

```diff
@@ -1,6 +1,7 @@
 include *.txt
 recursive-include wof/flask/templates *
 recursive-include wof/examples/flask/odm2/timeseries *
+recursive-include wof/examples/production_configs *
```

Comment:
If we remove that than no file from recursive-include `wof/examples/flask/odm2/timeseries` will be in the source tarball.
We could be more generic and use recursive-include `wof/examples/ *` instead while relying  on the `prune` lines alone to remove them.

Maybe the confusion is because depending on the `setuptools`/`pip` version you have you may get a different behavior regarding the `.py ` files (find all automatically or rely only on the `MANIFEST.in` file. So the prune lines are there to ensure that no `.py` that we don't want will be shipped. However, we have non-`.py` files and we do need to explicitly request them to be included in the tarball.)